### PR TITLE
Sb 3732: External preview for mimeType x-url

### PR DESCRIPTION
**Pull request by revathitarento**

### CODE REVIEW

**Comment by revathitarento** [comment]
@bvinayakumar : Please review the code

File: src/app/client/src/app/modules/learn/components/course-consumption/course-player/course-player.component.ts [hunk 1]
```diff
@@ -277,6 +279,10 @@ export class CoursePlayerComponent implements OnInit, OnDestroy {
       queryParams: { 'contentId': content.id },
       relativeTo: this.activatedRoute
     };
+    const contentDet = this.findContentById(content.id);
```
Comment (rajeevsathish) [inline src/app/client/src/app/modules/learn/components/course-consumption/course-player/course-player.component.ts]: what is the variable name ? can you have a name more descriptive name

File: src/app/client/src/app/modules/learn/components/course-consumption/course-player/course-player.component.ts [hunk 2]
```diff
@@ -124,7 +125,8 @@ export class CoursePlayerComponent implements OnInit, OnDestroy {
     private courseConsumptionService: CourseConsumptionService, windowScrollService: WindowScrollService,
     router: Router, public navigationHelperService: NavigationHelperService, private userService: UserService,
     private toasterService: ToasterService, private resourceService: ResourceService, public breadcrumbsService: BreadcrumbsService,
-    private cdr: ChangeDetectorRef, public courseBatchService: CourseBatchService) {
+    private cdr: ChangeDetectorRef, public courseBatchService: CourseBatchService, public externalUrlpreviewService:
+    ExternalUrlpreviewService) {
```
Comment (rajeevsathish) [inline src/app/client/src/app/modules/learn/components/course-consumption/course-player/course-player.component.ts]: why is the variable name is not in lowerCamelCase

Comment (revathitarento) [reply src/app/client/src/app/modules/learn/components/course-consumption/course-player/course-player.component.ts]: class name rule is to start with uppercase

File: src/app/client/src/app/modules/learn/components/course-consumption/course-player/course-player.component.ts (hunk 1 above)
Comment (revathitarento) [reply src/app/client/src/app/modules/learn/components/course-consumption/course-player/course-player.component.ts]: changed the name to playContentDetail

**Pull request closed without merging**